editors/code/src/commands.ts

```diff
@@ -947,10 +948,27 @@ export function openDocs(ctx: CtxInit): Cmd {
         const position = editor.selection.active;
         const textDocument = { uri: editor.document.uri.toString() };
 
-        const doclink = await client.sendRequest(ra.openDocs, { position, textDocument });
+        const doclinks = await client.sendRequest(ra.openDocs, { position, textDocument });
+
+        let doclink;
+        try {
+            if (
+                typeof doclinks.local === "string" &&
+                (await vscode.workspace.fs.stat(vscode.Uri.parse(doclinks.local)))
```

Comment:
We should also check that the type here is a file (and not a folder)

Reply:
Done. Also refactored the code a little to remove one layer of indentation.